src/movinginterquartilemean.h

```diff
@@ -0,0 +1,44 @@
+#ifndef MOVINGINTERQUARTILEMEAN_H
+#define MOVINGINTERQUARTILEMEAN_H
+
+#include <QLinkedList>
+#include <QQueue>
+
+//! Truncated Interquartile mean
```

Comment:
Sorry in advance for nit-pick code style comments. :)

Please remove the exclamation points in front of your comments.